index.d.ts

```diff
@@ -10,9 +10,21 @@ export interface Options {
 	readonly host?: string;
 }
 
-/**
- * Get an available TCP port number.
- *
- * @returns Port number.
- */
-export default function getPort(options?: Options): Promise<number>;
+declare const getPort: {
+  /**
+   * Get an available TCP port number.
+   *
+   * @returns Port number.
+   */
+  (options?: Options): Promise<number>;
+
+  /**
+   * Make a range of ports [from,to).
+   * @param from - First port of range(inclusive)
+   * @param to - Last port of range(exclusive)
```

Comment:
Why isn't `to` inclusive too? I think that would be more user-friendly.

Reply:
> That is like the question I wanted to discuss the most in this PR.
> I stick with programming convention of [from,to) ranges, as it's usually default and easier to use when you deal with multiple ranges, but maybe for this project we would want to do inclusive ranges.
> That is, with conventional ranges
> 
> ```
> intersects(range(a, b), range(b, c)) == false
> union(range(a, b), range(b, c)) == range(a, c)
> ```

Reply:
Whether or not you want an inclusive end range really depends on the situation. That's why in languages like Swift, you have syntax for both cases, `1...3` (inclusive) and `1..<3` (exclusive). Since we have to pick one, I think inclusive makes the most sense here.